docs/source/release-notes.rst

```diff
@@ -4,6 +4,50 @@ FiftyOne Release Notes
 .. default-role:: code
 .. include:: substitutions.rst
 
+FiftyOne 0.5.2
+--------------
+*Released August 26, 2020*
+
+App
+^^^
+- Added a label filter to the App that allows you to interactively explore your
+  labels, investigating things like confidence thresholds, individual classes,
+  and more, directly from the App
+
+Core
+^^^^
+- Added a :ref:`fiftyone quickstart <cli-fiftyone-quickstart>` command that downloads a small dataset,
+  launches the App, and prints some suggestions for exploring the dataset
+- Added support for multiple simultaneous FiftyOne processes. You can now
+  operate multiple App instances, Python shells, and/or CLI processes
```

Comment:
I think it would be worth noting the server port limitation (i.e. ports for additional servers need to be specified explicitly), although this point could get long if it's included.